changelog.md

```diff
@@ -3,6 +3,9 @@ when upgrading from a version of rust-sdl2 to another.
 
 ### v0.32
 
+[PR #824](https://github.com/Rust-SDL2/rust-sdl2/pull/824):
```

Comment:
For the changelog, technically, 0.32 is already released, so make a new category "v0.32.1 (unreleased)" and put those changes there instead. Otherwise, people will wonder why this rumble feature is not part of 0.32 on crates.io.

Other than than, looks good to me! I tested the update example locally on my computer with a Xbox 360 controller, works really well!

Reply:
Yeah, I wasn't so sure about that myself... What you proposed sounds good, I'm gonna do that.